src/docs/resources/inside-flutter.md

```diff
@@ -66,10 +66,10 @@ are a min and max width and a min and max height. During layout,
 the child determines its geometry by choosing a size within these bounds.
 After the child returns from layout, the parent decides the child's
 position in the parent's coordinate system<sup><a href="#a3">3</a></sup>.
-Notice that the child's layout cannot depend on the child's position
-because the child's position is not determined until after the child
-returns from layout. As a result, the parent is free to reposition
-the child without needing to recompute the child's layout.
+Note that the child's layout cannot depend on its position, as the position
```

Comment:
Hi, @Purukitto! You are completely right that the word "child" is overused in the original sentence. However, I think your edit could add some confusion as to what "it" is. It's better to be clear than to eject both extra uses of "child." How about:

Note that the child's layout can't depend on its position,
as the position is not determined until after the child
returns from layout. As a result, the parent is free to reposition
the child without needing to recompute its layout.